[ION-9593] Document HTTP Streaming does not work via a S/DLB (#523)
mulesoft/docs-runtime-manager: modules/ROOT/pages/dedicated-load-balancer-tutorial.adoc

For information about dedicated load balancers, see xref:cloudhub-dedicated-load-balancer.adoc[Dedicated Load Balancers].

---

[IMPORTANT]

> Load balancers do not support HTTP Streaming at this point. When a call is made directly to the worker HTTP streaming behaviour is demonstrated as expected (i.e An error response is received before the full attachment is uploaded). However, when you make a call to the load balancer, you don't receive the response until the full upload completes, which could take several minutes.

Reviewer: ```suggestion
Load balancers do not currently support HTTP streaming. When a call is made directly to the worker, HTTP streaming behavior occurs as expected (that is, an error response is received before the full attachment is uploaded). However, when you make a call to the load balancer, you don't receive the response until the full upload completes, which can take several minutes.
```

For information about dedicated load balancers, see xref:cloudhub-dedicated-load-balancer.adoc[Dedicated Load Balancers].

---

[IMPORTANT]

> The Loadbalancers do not support HTTP Streaming at this point. When a call is made directly to the worker HTTP streaming behaviour is demonstrated as expected (i.e An error response is received before the full attachment is uploaded)

Reviewer 2: ```suggestion
The DLBs don't support HTTP streaming.
When you make a call directly to the worker, you receive the expected error response before the full attachment is uploaded.
```

> However when a call is made to the Loadbalancer the response is not received until the full upload is complete (this could take many minutes).

Reviewer 2: ```suggestion
However, when you make a call to the DLB, you don't receive the response until the full upload completes, which could take several minutes.
```
